osl/preprocessing/batch.py

```diff
@@ -748,9 +762,15 @@ def run_proc_batch(
     if outdir is None:
         # Use the current working directory
         outdir = os.getcwd()
-    outdir = validate_outdir(outdir)
-    logsdir = validate_outdir(logsdir or outdir / "logs")
-    reportdir = validate_outdir(reportdir or outdir / "report")
+    if '{' in outdir and '}' in outdir:
+        # validate the parrent outdir - later do so for each subdirectory
+        tmpoutdir = validate_outdir(outdir.split('{')[0])
+        logsdir = validate_outdir(logsdir or tmpoutdir / "logs")
```

Comment:
I think if each subject has its own directory then the default directory to write log files to should be the subject specific directory: /sub-{...} rather than /sub-{...}/logs

Reply:
Currently, the logs would be written to the parent directory. See that `tmpoutdir` is the directory w/o the {sub-} business. As you say, we can choose to have the logs in the subject directory too, which I can see would make sense. Let me know if I should do so.